src/endpoints/http/collections/delete.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+
+function message(message) {
+  return { message };
+}
+/**
+ * Add the route to delete collection(s) for the users App.
+ * This route expects a db connection to be present on the request object.
+ * 
+ * @param  {Object} router - The express router.
+ */
+export function deleteCollections(router) {
+  router.post('/collections', (req, res, next) => {
+    const reqCollections = req.body.names;
+    if (!reqCollections) {
+      return res.status(400).send(message('names(s) of collection(s) is required'));
+    }
+    const allCollectionNames = req.db.getCollectionNames();
```

Comment:
I would consider the code from here is the business logic. I would prefer us to move the business logic out of the controller, even if sometimes the business logic is relatively simple. I think there are a few benefits:

1. loose coupling between controllers and the business logic. Make it easier to move the business logic if we need to.
2. easier to test the business logic.
3. keep things consistent. Make it looks like the code is written by one developer.

See https://github.com/feedhenry/fh-dataman/blob/master/src/endpoints/http/collections/index.js#L4 for example.